dev-support/docker-compose/scripts/start-namenode.sh

```diff
@@ -5,4 +5,5 @@ if [ $FORMAT_NAMENODE == true ]; then
 fi
 
 /opt/hadoop/hadoop-dist/target/hadoop-2.7.4/sbin/hadoop-daemon.sh --config /etc/hadoop --script hdfs start namenode
+/opt/hadoop/hadoop-dist/target/hadoop-2.7.4/sbin/httpfs.sh start
```

Comment:
It's better to start httpfs in its own container. Common case is to have a pool of httpfs replicas per cluster.